platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/signing/DefaultSelfEventSigner.java

```diff
@@ -47,6 +47,6 @@ public DefaultSelfEventSigner(@NonNull final KeysAndCerts keysAndCerts) {
     @Override
     public PlatformEvent signEvent(@NonNull final UnsignedEvent event) {
         final Signature signature = new PlatformSigner(keysAndCerts).sign(event.getHash());
-        return new PlatformEvent(event, signature.getSignatureBytes());
+        return new PlatformEvent(event, signature.getBytes().toByteArray());
```

Comment:
this copies the signature bytes unnecessarily

Reply:
done